src/onefetch/cli.rs

```diff
@@ -113,6 +114,23 @@ impl Cli {
                 ])
                 .help("Colors (X X X...) to print the ascii art."),
         )
+        .arg(
+            Arg::with_name("text-colors")
+                .short("t")
+                .long("text-colors")
+                .value_name("X")
+                .multiple(true)
+                .takes_value(true)
+                .max_values(6)
+                .possible_values(&[
+                    "0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14",
+                    "15",
```

Comment:
duplicate code with the possible_values of --ascii-colors, we should maybe put it in a variable.